counsel.el

```diff
@@ -3127,6 +3127,26 @@ otherwise continue prompting for tags."
 ;;;###autoload
 (defalias 'counsel-org-goto #'counsel-outline)
 
+(defcustom counsel-org-goto-all-outline-path-prefix nil
+  "When the value is 'file, include the file name (without
+directory) into the outline path candidate.
+
+When 'full-file-path, include the full file name.
+
+When 'buffer-name, include the buffer name."
+  :type '(choice (const :tag "None" nil)
+	         (const :tag "File name" file)
+	         (const :tag "Full file name" full-file-path)
```

Comment:
GNU projects use "path" to refer to lists of directories (e.g. `$PATH` and `load-path`), not full file names. Can you think of an alternative name, e.g. `file-name` and `nondirectory` or similar?

Reply:
I copied the symbols from `org-refile-use-outline-path`, but you are right, and since there's little reason for consistency with that other option, I changed the symbols to `file-name` and `file-name-nondirectory` instead. Thanks.